src/client/dfuse/dfuse_main.c

```diff
@@ -223,8 +223,11 @@ show_help(char *name)
 	printf("usage: %s -m=PATHSTR -s=RANKS\n"
 		"\n"
 		"	-m --mountpoint=PATHSTR	Mount point to use\n"
+		"	   --pool=UUID		pool UUID\n"
+		"	   --container=UUID	container UUID\n"
```

Comment:
looks like a bad merge here?
at least i don't understand why you list pool twice once with uuid and once with label & uuid..

Reply:
Yes, I think it was.  Will fix.